lib/skpm-link.js

```diff
@@ -72,9 +72,11 @@ if (!packageJSON.name) {
 console.log(chalk.dim('[1/1]') + ' 🔗  Symlinking the plugin ' + packageJSON.name + '...')
 
 try {
-  if (!fs.existsSync(path.join(pluginDirectory, packageJSON.name))) {
-    fs.mkdirSync(path.join(pluginDirectory, packageJSON.name))
+  if (fs.existsSync(path.join(pluginDirectory, packageJSON.name))) {
+    return console.log(chalk.red('error') + ' This plugin has already been linked.')
```

Comment:
that's not equivalent: the error is coming from `fs.symlinkSync`, not `fs.mkdirSync`. If you have a folder but no plugin inside, `skpm link` will now fail but it shouldn't.

I think we need another check for `fs.existsSync(path.join(pluginDirectory, packageJSON.name, packageJSON.main))`